src/main/java/com/emyus/QADBEditController.java

```diff
@@ -0,0 +1,46 @@
+package com.emyus;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import com.emyus.ca.CARequest;
+import com.emyus.ca.CAService;
+import com.emyus.ca.CorrectAnswer;
+import com.emyus.que.Question;
+import com.emyus.que.QuestionRequest;
+import com.emyus.que.QuestionService;
+@Controller
+public class QADBEditController {
+
+	@Autowired
+	private QuestionService questionService;
+
+	@Autowired
+	private CAService CAService;
+
+	@PostMapping("/qaeditconfirm2")
+	public String DBregister(@ModelAttribute("questions_id") int questions_id, @ModelAttribute("answer_id") int cas_id, @ModelAttribute("question") String question, @ModelAttribute("answer") String answer, Model model , QuestionRequest questionRequest, CARequest caRequest) {
+		System.out.println(questions_id);
+		System.out.println(cas_id);
```

Comment:
この辺りに
```suggestion
		System.out.println(cas_id);
		questionRequest.setId(1);
```
みたいに固定値のIDをsetしてみて更新されるか試してみてください。